models/CommentsManager.php

```diff
@@ -43,30 +44,41 @@ public function postComment($postId){
 
 
 
-	public function addComments(Comments $comments){
+	public function addComment($userId, $comment, $postId){
 
-		$req = $this->getBdd()->prepare('INSERT INTO comment(user_id, post_id, create_at, content, validation) VALUES ( :user_id, :post_id, NOW(), :content, false )');
+		$sql = "INSERT INTO comment ( user_id, post_id, create_at, content, validation, modified_at) VALUES ( :user_id, :post_id, now(), :content, false, null )";
+		$req = $this->getBdd()->prepare($sql);
+		$result = $req->execute([
+			':user_id' => $userId,
+			':post_id' => $postId,
+			':content' => $comment
+		]);
 
-		$req->blindValue(':user_id', $comments->getUserId(), PDO::PARAM_INT);
-		$req->blindValue(':post_id', $comments->getPostId(), PDO::PARAM_INT);
-		$req->blindValue(':content', $comments->getContent(), PDO::PARAM_STR);
+		$req->closeCursor();
 
-		$req->execute();
+		if ($result){
+			return true;
+		}else{
```

Comment:
Codacy found an issue: [The method addComment uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them.](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=5982252)